### ISSUE TITLE
Display Name of Current DJ / Show

### ISSUE THREAD

**Issue by dvdokkum** [WXYC/wxyc-ios-64#9]
There isn't much structured data for this, but if the DJ supplies a public "nickname" when they start their show that metadata should show up somewhere in the playlist json api.... i think?

**Comment by swardlincoln** [comment]
@dvdokkum I was looking at [this url ](http://wxyc.info/playlists/recentEntries?v=2&n=15) to see if there was any way to parse out DJ info, but I did find anything. Was the URL I was looking at the source of the text data that is populating the app?

**Comment by dvdokkum** [comment]
Yes, that's what we're using currently. More generically, http://www.wxyc.info/playlists/recentEntries (and the v2 version). You're right, it looks like that json feed does not include the show start/stop entry that shows in the regular schedule. @jakebromberg you have thoughts on this, i believe?

**Comment by jakebromberg** [comment]
We either need to work with Tim Ross to extend his API, or scrape the playlist HTML to get this information.

**Comment by swardlincoln** [comment]
Does swift have a scraping lib? @jakebromberg

**Comment by swardlincoln** [comment]
Ah...okay. So basically it would be best to reach out to Tim and have him extend the API?

**Comment by jakebromberg** [comment]
Yeah. I'm in town starting Saturday. I'm going to see if he wants to get lunch.
